XmlFormatter/src/Settings/Provider/DataStructure/SerializableSettingContainer.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections.Generic;
+
+namespace XmlFormatter.src.Settings.Provider.DataStructure
+{
+    /// <summary>
+    /// A saveable settings container
+    /// </summary>
+    public class SerializableSettingContainer
+    {
+        /// <summary>
+        /// All the scopes in this container
+        /// </summary>
+        private List<SerializableScope> scopes;
+
+        /// <summary>
+        /// Public access to all the scopes in this container
+        /// </summary>
+        public List<SerializableScope> Scopes
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this an auto-implemented property and remove its backing field.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5052706)